welcome/index.html

```diff
@@ -0,0 +1,352 @@
+            Replace the host with the address of the machine the experimental platform is running on.
+          </p>
+        </li>
+        <li>
+          <h4>Connect to the Philips Hue</h4>
+          <p class="primary">
+            Controlling the Philips Hue lightbulb couldn't be easier, just use an http request to control the lightbulb:
+          </p>
+          <code>PUT: hue.local/api/:token/lights/:lights_id/state</code>
+        </li>
+        <li>
+          <h4>See it in action</h4>
+          <p class="primary">
+            We prepared an example app to demonstrate all the different parts acting together to create an actually useful app: Change the color of the lightbulb depending on the CO₂ levels. <a href="https://github.com/experimental-platform/example-co2-to-hue">Check it out on github </a>.
+          </p>
+          <div>
+            <img src="img/co2demo.gif" alt="co2demo" class="tutorial__screenshot">
+          </div>
+          <code>git clone https://github.com/experimental-platform/example-co2-to-hue.git && cd example-co2-to-hue</code>
+        </li>
+        <li>
+          <h4>Deploy it</h4>
+          <p class="primary">Now deploy the app from your computer:</p>
+          <code>git remote add platform ssh://dokku@your-box.local:8022/example-co2-to-hue
+git push platform master</code>
+        </li>
+        <li>
+          <h4>That's it! Your app is now available on the internet and within your local network.</h4>
+          <p class="primary">Locally via <code>http://your-name.local/example-co2-to-hue/</code> and remotely via <code>https://your-name.protonet.info/example-co2-to-hue/</code></p>
+        </li>
+      </ol><!-- /.tutorial__steps -->
+
+      <div class="order-hardware">
+        <img class="order-hardware__illustration" src="img/order-hardware.jpg" alt="order hardware">
+        <p class="order-hardware__note">If you like to try the example on real hardware, you can for example order the modules via the links below.</p>
+        <a class="button-interaction" href="http://www.co2meter.com/collections/co2-sensors/products/s8-miniature-co2-sensor" target="_blank">Order the CO₂ Sensor
+          <span class="order-hardware__store">co2meter.com</span>
+        </a>
+        <a class="button-interaction" href="http://www.amazon.com/dp/B014H2P4KW/ref=sr_ph?ie=UTF8&amp;qid=1452304434&amp;sr=1&amp;keywords=philips+hue+ambiance+white+and+color" target="_blank">Order the Smartlight
+          <span class="order-hardware__store">amazon.com</span>
+        </a>
+        <a class="button-interaction" href="http://www.newegg.com/Product/Product.aspx?Item=N82E16856501013" target="_blank">Order the Mini PC
+          <span class="order-hardware__store">newegg.com</span>
+        </a>
+        <p class="order-hardware__note">
+          These links lead to external stores so you can quickly buy these devices and reproduce this project in real life (delivery might take more than 5 minutes though :) ).
```

Comment:
I would remove the ``:)`` - our font is not best suited for ascii smilies and the text looks weird with it